### PR TITLE
20247 Fixed GCP UI Redirection

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#20247

*Description of changes:*
Fixed GCP redirection to Create and Edit UIs

Sev, just an FYI, here are the changes that've been done to 1Pass by Argus that I'm using here:
MicrosoftTeams-image (12)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-631-olv84ct4.web.app

File: devops/vaults.env [hunk 1]
```diff
@@ -8,10 +8,10 @@ VUE_APP_ADDRESS_COMPLETE_KEY="op://canadapost/$APP_ENV/address-key/ADDRESS_COMPL
 VUE_APP_REGISTRY_HOME_URL="op://web-url/$APP_ENV/registry/REGISTRY_HOME_URL"
 VUE_APP_AUTH_WEB_URL="op://web-url/$APP_ENV/auth-web/AUTH_WEB_URL"
 VUE_APP_BUSINESSES_URL="op://web-url/$APP_ENV/business/BUSINESSES_URL"
-VUE_APP_DASHBOARD_URL="op://web-url/$APP_ENV/business/DASHBOARD_URL"
+VUE_APP_DASHBOARD_URL="op://web-url/$APP_ENV/business-le/DASHBOARD_URL"
 VUE_APP_SITEMINDER_LOGOUT_URL="op://web-url/$APP_ENV/siteminder/SITEMINDER_LOGOUT_URL"
-VUE_APP_BUSINESS_CREATE_URL="op://web-url/$APP_ENV/business-create/BUSINESS_CREATE_URL"
-VUE_APP_BUSINESS_EDIT_URL="op://web-url/$APP_ENV/business-edit/BUSINESS_EDIT_URL"
+VUE_APP_BUSINESS_CREATE_URL="op://web-url/$APP_ENV/business-create-le/BUSINESS_CREATE_URL"
+VUE_APP_BUSINESS_EDIT_URL="op://web-url/$APP_ENV/business-edit-le/BUSINESS_EDIT_URL"
```
Comment (severinbeauvais) [inline devops/vaults.env:14]: Is this change for eventual deployments using CD scripts? I think you don't need to do this for the "gcbrun" (cloudbuild) environments.

Actually, what are your environments?

Comment (argush3) [reply devops/vaults.env:14]: Dev for sure and I am in the process of pushing legal api in GCP to test.  So i'm thinking we will have legal name UIs running in test too so others can test.

Comment (severinbeauvais) [reply devops/vaults.env:14]: An actual Dev environmemnt (using a CD script), or a cloudbuilt environment ("gcbrun")?

Comment (JazzarKarim) [reply devops/vaults.env:14]: Isn't it that the actual Dev environment grabs the values from 1pass while the cloud build environment grabs from the `.env.example`? I was thinking to update both to tackle both scenarios.

Comment (severinbeauvais) [reply devops/vaults.env:14]: ^^ Yes, you're right.

👍 to tackling both scenarios, but I'm still curious about your environments :)

**Review by severinbeauvais — APPROVED** [review]

File: devops/vaults.env (hunk 1 above)
Comment (argush3) [reply devops/vaults.env:14]: I'm still talking to Patrick about this.  Ideally there is just one set of environments but we are still waiting until everyone can get on legal name branch as you know.

We'll probably try to hold on in just using dev for legal name UIs for now.

**Review by argush3 — APPROVED** [review]

File: devops/vaults.env (hunk 1 above)
Comment (severinbeauvais) [reply devops/vaults.env:14]: Are you actually in "dev"? Or a cloudbuild environment?

Comment (JazzarKarim) [reply devops/vaults.env:14]: We're actually in Dev. We haven't been using the cloudbuild (gcbrun) URLs Sev.

**Pull request merged**